torch/csrc/jit/passes/shape_analysis.cpp

```diff
@@ -532,8 +531,6 @@ bool PropagateTensorShapeOnNode(Node * node, bool insert_expands) {
     "aten::mul(Tensor self, Tensor other) -> Tensor",
     "aten::div(Tensor self, Tensor other) -> Tensor",
     "aten::pow(Tensor self, Tensor exponent) -> Tensor",
-    "aten::min(Tensor self, Tensor other) -> Tensor",
-    "aten::max(Tensor self, Tensor other) -> Tensor",
```

Comment:
Why did we disable these? (and `exp`)?

Reply:
They were repeated, so I removed duplicates.